api/apps/api/src/modules/access-control/projects-acl/project-acl.controller.ts

```diff
@@ -0,0 +1,80 @@
+import { apiGlobalPrefixes } from '@marxan-api/api.config';
+import {
+  Controller,
+  Get,
+  Req,
+  Param,
+  ParseUUIDPipe,
+  UseGuards,
+  ForbiddenException,
+  Delete,
+  Patch,
+  Body,
+} from '@nestjs/common';
+import { JwtAuthGuard } from '@marxan-api/guards/jwt-auth.guard';
+import { ProjectAclService } from './project-acl.service';
+import { RequestWithAuthenticatedUser } from '@marxan-api/app.controller';
+import { ApiBearerAuth, ApiTags } from '@nestjs/swagger';
+import { UsersProjectsApiEntity } from '@marxan-api/modules/projects/control-level/users-projects.api.entity';
+import { UpdateUserRoleinProject } from './dto/update-user-role-project.dto';
+
+@UseGuards(JwtAuthGuard)
+@ApiBearerAuth()
+@ApiTags('Projects Users')
+@Controller(`${apiGlobalPrefixes.v1}/projects`)
```

Comment:
A couple of thoughts there:

1. `/projects` - it will most likely override other controller (not sure Nest can "merge" routes)
2. `/projects` isn't the resource we change in REST terms (we change privileges/roles)
3. Sub-modules should be rather "clean" in terms of functionality and not bind to particular transport layer. Think of having them "pure" for doing one thing - access to roles within a project in this case
4. Thus, I think it *could* be good to move it to `access-control` or even outside of the whole `access-control` module. (as controller is "only" a way to access this module capabilities for system consumers)